.grabl/automation.yml

```diff
@@ -60,7 +60,7 @@ build:
         export ARTIFACT_USERNAME=$REPO_GRAKN_USERNAME
         export ARTIFACT_PASSWORD=$REPO_GRAKN_PASSWORD
         bazel run @graknlabs_dependencies//distribution/artifact:create-netrc
-        bazel test //test/behaviour/connection/... --test_output=errors
+        bazel test //test/behaviour/connection/... --test_output=errors --jobs=1
```

Comment:
Is this set in order to avoid creating too many databases concurrently, causing memory overflow? @lriuui0x0

Reply:
Spoke verbally with @lriuui0x0 and the answer is yes